### DISCUSSION TITLE
Generate documentation for private packages?

### DISCUSSION THREAD

**Discussion by busslina** [GregoryConrad/rearch-dart#40] [Off topic]
I want to generate documentation web pages for my private packages.
Do you know if it is possible? Like this: https://pub.dev/documentation/rearch/latest/

**Comment by busslina** [comment]
Just found this: https://pub.dev/packages/dartdoc

**Reply by GregoryConrad** [reply]
Yup, I believe `dart doc` is exactly what `pub.dev` runs for package authors automatically (and produces the documentation page you linked).
